user-journey-tests/page-objects/destination/destinationAnswersPage.js

```diff
@@ -1,20 +1,25 @@
-import { Page } from '../page.js'
+import { AnswersBasePage } from '../base-pages/answersBasePage.js'
 
 const pageHeadingAndTitle =
   'Check your answers before you continue your application'
+const changeLinks = {
+  destinationType: '[data-testid="destinationType-change-link"]',
+  parishNumber: '[data-testid="destinationFarmCph-change-link"]',
+  address: '[data-testid="destinationFarmAddress-change-link"]',
+  maxAnimals: '[data-testid="howManyAnimalsMaximum-change-link"]',
+  reason: '[data-testid="reasonForMovement-change-link"]',
+  additionalInfo: '[data-testid="additionalInfo-change-link"]',
+  over75: '[data-testid="movingMoreThan75Animals-change-link"]',
+  halfHerd: '[data-testid="movingMoreThanHalfExistingHerd-change-link"]'
+}
 
-class DestinationAnswersPage extends Page {
+class DestinationAnswersPage extends AnswersBasePage {
   pagePath = 'destination/check-answers'
   pageHeading = pageHeadingAndTitle
   pageTitle = pageHeadingAndTitle
 
-  // Change links
-  get changeDestinationLink() {
-    return $('[data-testid="destinationType-change-link"]')
-  }
-
-  get destinationValue() {
-    return $$('.govuk-summary-list__value')[0]
+  constructor() {
```

Comment:
if nothing else going in constructor, you can delete it and will be picked up by inheritence